modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -3475,5 +3475,19 @@ public String getIgnoreFilePathOverride() {
      */
     public void setIgnoreFilePathOverride(final String ignoreFileOverride) {
         this.ignoreFilePathOverride = ignoreFileOverride;
+
```

Comment:
@wing328 looks like a bad merge, maybe, causing CI to fail. I think this is missing the closing semicolon.